src/main/java/io/jenkins/plugins/ml/IPythonBuilder.java

```diff
@@ -106,7 +106,7 @@ public IPythonBuilder(
     @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
     public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath ws, @Nonnull Launcher launcher, @Nonnull TaskListener listener) throws AbortException {
         try {
-
+//            LOGGER_3.setLevel(Level.OFF);
```

Comment:
:+1: we can uncomment once we understand what's going on via ZEPPELIN-4569.

Don't forget if we do set the level here, we must confirm whether that changes globally or not. If it does indeed disable the level globally, that's really dangerous, so we would need to reset the level to its previous value.